packages/frontend/core/src/components/page-list/scoped-atoms.tsx

```diff
@@ -191,14 +191,6 @@ export const groupsAtom = atom(get => {
 
   if (groupBy === false) {
     groupBy = undefined;
-  } else if (groupBy === undefined) {
```

Comment:
I forgot why I added this else branch. It seems groupBy should never be false now. Maybe you can remove `false` from groupBy's type